x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/all/columns.tsx

```diff
@@ -182,14 +186,23 @@ export const getColumns = ({
       field: 'enabled',
       name: i18n.COLUMN_ACTIVATE,
       render: (value: Rule['enabled'], item: Rule) => (
-        <RuleSwitch
-          data-test-subj="enabled"
-          dispatch={dispatch}
-          id={item.id}
-          enabled={item.enabled}
-          isDisabled={hasNoPermissions}
-          isLoading={loadingRuleIds.includes(item.id)}
-        />
+        <EuiToolTip
+          position="top"
+          content={
+            item.type === 'machine_learning' && !hasMlPermissions
```

Comment:
There's an `isMlRule` helper in `common` to prevent this string from spreading through our codebase.

Reply:
Yahs! Thank you -- was looking for that 🙂